src/main/java/Duke.java

```diff
@@ -1,10 +1,230 @@
+                        if (line.substring(2).startsWith("1")) {
+                            taskList.get(taskList.size() - 1).setDone();
+                        }
+                    } else {
+                        String description = line.substring(4);
+                        Todo newTask = new Todo(description);
+                        taskList.add(newTask);
+                        if (line.substring(2).startsWith("1")) {
+                            taskList.get(taskList.size() - 1).setDone();
+                        }
+                    }
+                }
+            }
+        } catch (Exception e) {
+            System.out.println("sth");
+        }
+    }
+
+
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        readFileContents("lines.txt");
+
+        Scanner in = new Scanner(System.in);
+
+        System.out.println("Hello Bbygirl! I'm Your Boyfriend <3");
+        System.out.println("How can I help you today? ;)");
+        String t = in.nextLine();
+
+        while (!t.equals("bye")) {
+            if (t.equals("list")) {
+                printTaskList();
+            } else if (t.startsWith("done")) {
+                try {
+                    markDone(t);
+                } catch (IllegalDoneException e) {
+                    System.out.println("You need to input a correct number BB... ;'( try typing again");
+                }
+            } else if(t.startsWith("delete")) {
+                try {
+                    deleteTask(t);
+                } catch (IllegalDoneException e) {
+                    System.out.println("You need to input a correct number BB... ;'( try typing again");
+                }
+            } else {
+                try {
+                    typeOfTask(t);
+                } catch (IllegalTaskException e) {
+                    System.out.println("You have a typo BB.. ;'( try typing again");
+                } catch (IndexOutOfBoundsException e) {
+                    System.out.println("You need to add an item BB.. ;'( try typing again");
+                } catch (InvalidDeadlineFormat e) {
+                    System.out.println("You need to input deadline with '/by' ... ;'( try typing again");
+                } catch (InvalidEventFormat e) {
+                    System.out.println("You need to input event with '/at' ... ;'( try typing again");
+                }
+            }
+            t = in.nextLine();
+        }
+        try {
+            writeToFile("lines.txt");
+        } catch (IOException e) {
+            System.out.println("Something went wrong: " + e.getMessage());
+        }
+        System.out.println("Goodbye. I will miss you sooo much :(");
```

Comment:
You should minimize the size of the main method by splitting the different functionalities into their own methods. By reducing the size of the method and ensuring each method does one functionality it is supposed to do, your code is easier to maintain, scale and debug.